.buildkite/pipeline.yml

```diff
@@ -33,7 +35,7 @@ steps:
    commands:
    - eval "$SET_VERSION"
    - cd docker
-   - nix run -f.. pkgs.docker -c ./docker-static-build.sh
+   - nix run -f.. pkgs.podman pkgs.conmon pkgs.runc pkgs.shadow -c ./docker-static-build.sh
```

Comment:
podman shouldn't need `nix run`, it's available globally on our CI server, and it's better to use the same version as the server provides

Reply:
the problem is that `podman` isn't available when running buildkite job for some reason, see:
https://buildkite.com/serokell/tezos-packaging/builds/540#b82f4312-e7dd-4c6f-8c4e-17d80c7087a8/63-68

Reply:
maybe buildkite-agent service needs to be restarted :thinking:

Reply:
Yeah, try restarting it, it might have the old `PATH` set

Reply:
Nope, that doesn't help:(

Reply:
Maybe there is no PATH in the service configuration at all? :thinking:

Reply:
there is `PATH`, but there is no `podman` in it even after restart

Reply:
https://github.com/serokell/serokell-profiles/blob/master/profiles/servers/jupiter/default.nix#L280 :coolstory:

Reply:
_Why do computers have to be so hard._ So we need to add podman to the list, enabling a package globally works for gitlab-runner but not for buildkite

Reply:
https://github.com/serokell/serokell-profiles/pull/536

Reply:
works now :tada: